src/containers/lineup/CardLineup.tsx

```diff
@@ -51,23 +51,35 @@ const DesktopCardContainer = ({
   )
 }
 
+const EmptyMobileCard = () => (
+  <div className={styles.mobileCardContainer}>
+    <div className={styles.emptyMobileCard} />
+  </div>
+)
+
 const MobileCardContainer = ({ cards, containerClassName }: OwnProps) => {
-  const emptyMobileCard = cards.length % 2 === 1
+  let emptyCards: ReactElement | null = null
+  if (cards.length === 1) {
+    emptyCards = (
+      <>
+        <EmptyMobileCard />
+        <EmptyMobileCard />
+      </>
+    )
+  } else if (cards.length === 2) {
+    emptyCards = <EmptyMobileCard />
+  } else if (cards.length % 2 === 1) {
+    emptyCards = <EmptyMobileCard />
+  }
+
```

Comment:
You could split this into a function to separate the logic and avoid mutating `emptyCards`:

```suggestion
const getEmptyCards = (cardLength: number) => {
  if (cardsLength === 1) {
    return (
      <>
        <EmptyMobileCard />
        <EmptyMobileCard />
      </>
    )
  } else if (cardsLength === 2) {
    return <EmptyMobileCard />
  } else if (cards.length % 2 === 1) {
    return <EmptyMobileCard />
  }
}
```

Reply:
totally can do, was alternating between the two, thanks!